ovirtlib4/system_service.py

```diff
@@ -151,44 +151,33 @@ def get(self, func, wait_for=None, *args, **kwargs):
         return func(*args, **kwargs)
 
 
-class CollectionService(RootService):
+class OvirtService(RootService):
     """
-    Abstract class, represent an oVirt collection
+    Abstract class, represent an oVirt service with its manage 'type' object
     """
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         self._service = NotImplementedError
-        self._entity_service = NotImplementedError
         self._entity_type = NotImplementedError
 
     @property
     def service(self):
-        """ Return the main collection service e.g.: vms_service(), hosts_service() """
+        """ Return the ovirt service """
         return self._service
 
     @service.setter
     def service(self, service):
         """ The collection service must be set by the inherit class """
         self._service = service
 
-    @property
-    def entity_service(self):
-        """ Return the service of an individual entity of the collection """
-        return self._entity_service
-
-    @entity_service.setter
-    def entity_service(self, entity_service):
-        """ The _entity_service must be set by the inherit class """
-        self._entity_service = entity_service
-
     @property
     def entity_type(self):
```

Comment:
```suggestion
    @abstractmethod
    def entity_type(self):
```

See [link](https://docs.python.org/3.6/library/abc.html#abc.abstractmethod)

Reply:
same ^